### PR TITLE
Update settings.yml for dev-ko branch protection

### PR DESCRIPTION
**Pull request by seokho-son**
This PR updates

settings.yml for `dev-ko` branch protection rule and approvers.

I am not sure this configuration works, but we can check after this PR's been merged.

### CODE REVIEW

**Comment by CathPag** [comment]
@seokho-son, if this is the Korean branch, why is Edson's name on it? A mistake?
Screen Shot 2022-01-04 at 8 07 47 AM

**Comment by seokho-son** [comment]
@CathPag 
It is the existing setting for PT branch :)

This PR just adds configuration for Korean only to the existing setting.

But after I check all configurations and procedures work properly for Korean localization, I am going to update the current setting for PT as well. :)

**Review by CathPag — APPROVED** [review]
LGTM

**Comment by seokho-son** [comment]
@CathPag I would like to check if I need to wait for further approval on this PR or if I can merge this PR myself as a maintainer. :)

**Comment by CathPag** [comment]
Please go ahead :)

**Pull request merged**
